data/static_data.rb

```diff
@@ -29,6 +30,16 @@ class StaticData
   PALLADIUM_SERVER = 'palladium.teamlab.info'
   POSITIVE_STATUSES = %w[passed passed_2].freeze
 
+  FORMAT_NUMBERS = {
```

Comment:
I think some description is needed - why those numbers
For example link to `core` source lines or maybe some source of this number (Like "Those data from Oleg Korshul`)

Reply:
I also think since those FORMAT_NUMBER are used only in XmlParams - this constant should be placed in this file, and not in static data